.cirrus.yml

```diff
@@ -0,0 +1,19 @@
+env:
+  CIRRUS_CLONE_DEPTH: 0 # Unshallow clone to obtain proper GIT_VERSION
+
+freebsd_task:
+  matrix:
+    - name: FreeBSD 12.2 (snapshot)
+      freebsd_instance:
+        image_family: freebsd-12-1-snap
+        cpu: 8
+        memory: 8G
+  env:
+    CCACHE_MAXSIZE: 300M # 3x clean build, rounded
+    CCACHE_DIR: /tmp/ccache_dir
+  ccache_cache:
+    folder: /tmp/ccache_dir
+  packages_cache:
+    folder: /var/cache/pkg
```

Comment:
I did try to cache `/usr/local` (install prefix) +`/var/db/pkg` (metadata) but it was fragile. Instances could be updated externally, and at least `python2.7` couldn't be cleanly overwritten:
```
Downloaded 590Mb.
Cache hit for install_prefix-<hash>!
Failed to unarchive install_prefix cache because of /usr/local/bin/python2.7: creating new file: open /usr/local/bin/python2.7: text file busy! Retrying...
Downloaded 590Mb.
Cache hit for install_prefix_<hash>!
```